modules/data.atmosphere/R/download.NARR_site.R

```diff
@@ -86,8 +86,8 @@ download.NARR_site <- function(outfolder,
 #' @param file Full path to target file
 #' @param lat_nc `ncdim` object for latitude
 #' @param lon_nc `ncdim` object for longitude
-#' @param verbose 
-#' @return List of NetCDF variables in data. Creates NetCDF file containing 
+#' @param verbose
```

Comment:
```suggestion
#' @param verbose logical: ask`ncdf4` functions to be very chatty while they work?
```

Documenting arguments is always good, but the reason we need this now is kind of 🙄: With the trailing space included, Roxygen was previousy complaining about "argument items with no description" and we ignored it in Rcheck_reference.log. Without the trailing space, Roxygen complains instead about "undocumented arguments" and causes a CI failure because the ignore-preexisting-errors script only knows how to skip exact test matches.

After fixing this we should delete the "with no description" from Rcheck_reference.log (currently lines 420-421).